network/rpc/server/src/bin/client.rs

```diff
@@ -0,0 +1,82 @@
+//! Example of using jsonrpsee client.
+//!
+//! Create a nexus project, build it and get path to elf file, for example
+//! ```sh
+//! realpath target/riscv32i-unknown-none-elf/debug/my_program
+//! ```
+//!
+//! Run the client with `cargo run -r --bin client -- <PATH TO ELF>`
+
+use std::env;
+use std::time::Duration;
+
+use jsonrpsee::ws_client::{WsClient, WsClientBuilder};
+use nexus_prover::types::IVCProof;
+use nexus_rpc_common::ArkWrapper;
+use nexus_rpc_traits::RpcClient;
+
+use tracing::Level;
+use tracing_subscriber::{
+    filter, fmt::format::FmtSpan, layer::SubscriberExt, util::SubscriberInitExt,
+};
+
+// type used by the RPC.
+type RpcProof = ArkWrapper<IVCProof>;
+
+#[tokio::main]
+async fn main() {
+    setup_logger();
+
+    // path to elf
+    let args: Vec<String> = env::args().collect();
```

Comment:
Why not use `clap`?

Reply:
Unneeded dependency for an example code.

Reply:
Sounds good, treating this as demo and not production code.